src/user/serializers.py

```diff
@@ -708,10 +759,14 @@ def _group_and_serialize_actions(self):
                     data["paper_id"] = thread_paper.id
                 elif thread_post:
                     data["parent_content_type"] = "post"
-                    data["paper_title"] = (
+                    data[
+                        "paper_title"
+                    ] = (
                         thread_post.title
                     )  # paper_title instead of post_title for symmetry on the FE
-                    data["paper_id"] = (
+                    data[
+                        "paper_id"
+                    ] = (
```

Comment:
Weird, our formatters are off. I remember this was just changed by one of my previous commits...

Reply:
My editor has it the original way but the pre commit changes it. Haven't looked into why.

Reply:
Got it, we should take a look at that.